packages/testring-cli/src/index.ts

```diff
@@ -79,23 +80,25 @@ export const runCLI = (argv: Array<string>) => {
     const args = yargs.parse(argv);
     const command = args._[2];
 
-    let commandExecution;
-
     switch (command) {
         case 'run':
-            commandExecution = runTests(argv, process.stdout);
+            runTests(argv, process.stdout).catch((exception) => {
```

Comment:
Why did you move `catch(..)` back to command call? I think solution from master was better